stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     {{- end }}
     port: https
     scheme: https
+{{- if .Values.kubeApiServer.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+{{ toYaml .Values.kubeApiServer.serviceMonitor.metricRelabelings | indent 6 }}
```

Comment:
For consistency should you have it `indent 4`?

Reply:
6 is right for this